contracts/upgradeable_contracts/arbitrary_message/BasicAMB.sol

```diff
@@ -17,6 +33,7 @@ contract BasicAMB is BasicBridge {
         require(_gasPrice > 0);
         require(_requiredBlockConfirmations > 0);
 
+        _setChainIds(_sourceChainId, _destinationChainId);
```

Comment:
I thought it is reasonable to store chain ids in `bytes` rather than in a pair of `uint256`, since for short bytes blobs only one storage slot will be occupied.
I compared gas costs for both approaches, seems that using `uintStorage` is still a bit better solution.

I updated the contracts with the proposed approach.

Reply:
Yeah, I was considering that a short blob could take one slot and that's why we will have just one `SLOAD` operation but was also optimistic that operations with bytes set packing/unpacking could take more resources :)